### ISSUE TITLE
Zirnox reactor is blow up after enter the world

### ISSUE THREAD

**Issue by SannUlya** [HbmMods/Hbm-s-Nuclear-Tech-GIT#698]
When I enter my minecraft world. I was surprised that my ZIRNOX Reactor accidentally exploded.

I tried to log back in and quickly shut down the ZIRNOX reactor. I made a backup of my world and will investigate the case. I turned on my zirnox reactor then logged out, and when I logged in again I checked the gui and found the cause of the explosion because of the super dense vapor that accumulates. Though I have installed the steam pipe correctly. I turned off my ZIRNOX reactor, then checked to see if any steam pipes had been blown off and to no avail. I again checked my ZIRNOX Reactor and found that the water vapor had been transferred to the turbine properly. I restarted the ZIRNOX reactor and everything seems to be working fine. I logged out and logged in again this problem happened again.

In conclusion, the zirnox steam pipe has a delay when entering the game. Previously I did not experience an explosion when entering the world while the ZIRNOX reactor was still running.

**Comment by Creeper-banner** [comment]
The reason of the delay may be that the chunks in your world didn't load at the same time. The chunk with the reactor loaded first, and the reactor exploded because of too much steam before the chunk with turbine loaded.
Try to put your reactor and other stuffs in one chunk.

**Comment by Creeper-banner** [comment]
The reason of the delay may be that the chunks in your world didn't load at the same time. The chunk with the reactor loaded first, and the reactor exploded because of too much steam before the chunk with turbine loaded.
Try to put your reactor and other stuffs in one chunk.

**Comment by Creeper-banner** [comment]
The reason of the delay may be that the chunks in your world didn't load at the same time. The chunk with the reactor loaded first, and the reactor exploded because of too much steam before the chunk with turbine loaded.
Try to put your reactor and other stuffs in one chunk.

**Comment by SannUlya** [comment]
> The reason of the delay may be that the chunks in your world didn't load at the same time. The chunk with the reactor loaded first, and the reactor exploded because of too much steam before the chunk with turbine loaded. Try to put your reactor and other stuffs in one chunk.

Thanks for your opinion, this is make sense. I will try it
